exercises/concept/calculator-conundrum/.docs/instructions.md

```diff
@@ -22,7 +22,15 @@ The main method for implementation in this task will be the (_static_) `SimpleCa
 
 ## 2. Handle illegal operations
 
-Any other operation symbol should throw the `ArgumentOutOfRangeException` exception. If the operation argument is an empty string, then the method should throw the `ArgumentException` exception. When `null` is provided as an operation argument, then the method should throw the `ArgumentNullException` exception. This functionality and handling of operations should be contained in a `try` block.
+Any other operation symbol should throw the `ArgumentOutOfRangeException` exception. If the operation argument is an empty string, then the method should throw the `ArgumentException` exception. When `null` is provided as an operation argument, then the method should throw the `ArgumentNullException` exception.
+
+```csharp
+SimpleCalculator.Calculate(100, 10, "-"); // => throws ArgumentOutOfRangeException
+
+SimpleCalculator.Calculate(8, 2, ""); // => throws ArgumentException
+
+SimpleCalculator.Calculate(58, 6, null); // => throws ArgumentNullException
+```
 
 ## 3. Handle the thrown DivideByZero exceptions
```

Comment:
Maybe rename this to:

```suggestion
## 3. Handle division by zero
```

Note: the hints also need to be updated accordingly

Reply:
Good suggestion. I also think we should not mention `DivideByZeroException` in the task instruction and let the student figure that out themselves. Maybe we should help them out by pointing to it from the hints?

Reply:
Yeah I like that a lot. Great suggestions!